rllib/utils/tf_utils.py

```diff
@@ -297,6 +297,21 @@ def huber_loss(x: TensorType, delta: float = 1.0) -> TensorType:
     )
 
 
+@PublicAPI
+def l2_loss(x: TensorType) -> TensorType:
```

Comment:
Tensorflow also [has this loss](https://www.tensorflow.org/api_docs/python/tf/nn/l2_loss)! If there is no particular reason here for us to not use it, I think we should.

Reply:
I looked at it. tensorflow one you need to differentiate between tf1 and tf2 it seems.
I figured that's why we want to roll our own.